### ISSUE TITLE
Same problem with fine tuning. 

### ISSUE THREAD

**Issue by JinnyZhao** [Arsey/keras-transfer-learning-for-oxford102#13]
Your issue is  that  I'm trying to adopt this approach for multiclass classification. And the first step when we're training the top model from bottlenecks works fine and I have 76% accuracy for 102 classes. But when I proceed to fine-tune step, the model with each next epoch decreases in accuracy. So if at the end of 1st epoch of fine-tune I have about 45% of accuracy, on the 2nd epoch I'm getting 40%, etc. 

Same problem with fine tuning. Im trying now and model decreased in accuracy from 70% to 40%.
 I'm trying to find the issue for a few weeks but no results. Whether you have  resolved it，thanks！

**Comment by Arsey** [comment]
I've simplified the fine-tuning process and not it should work. There's also no such steps like getting bottlenecks and top-model training.
